vulnerability-classes/injection/SSTI-Razor.bcheck

```diff
@@ -0,0 +1,27 @@
+metadata:
+    language: v2-beta
+    name: ".NET Razor SSTI Injection"
+    description: "Inserts a .NET Razor SSTI payload into each parameter to detect suspicious input evaluation"
+    author: "alp1n3.eth"
+    tags: "active", "injection", "template", "template injection", "scan", "ssti", "razor"
+# reference 1: https://clement.notin.org/blog/2020/04/15/Server-Side-Template-Injection-(SSTI)-in-ASP
```

Comment:
This link 404's for me.

Reply:
Updated BCheck with the correct link -- apologies, don't know why it got caught off there. I was fixing some editor issues surrounding wordwrap so it was probably that. Link is included now to: https://clement.notin.org/blog/2020/04/15/Server-Side-Template-Injection-(SSTI)-in-ASP.NET-Razor/